packages/wallet/src/redux/protocols/concluding/readme.md

```diff
@@ -0,0 +1,61 @@
+# Concluding Protocol
+
+The purpose of this protocol is to resign a channel, i.e. to move to a conclude state.
+It covers:
+
+- Checking to see if it is the player's turn and explaining they can't resign if not
+- Asking user to confirm the resignation (probably displaying the current outcome)
+- Formulating the conclude state and sending to the opponent
+- Waiting for a conclude from the opponent
+- Acknowledge channel concluded (giving the option to defund)
+
+Out of scope (for the time being):
+
+- Giving the option to launch a challenge if the conclude doesn't arrive
+- Allowing the user to not choose to not defund
+
+## State machine
+
+The protocol is implemented with the following state machine
+
+```mermaid
+graph TD
+  S((Start)) --> E{Channel Exists?}
+  E --> |No| F1((Failure))
```

Comment:
If this is going to be a top-level protocol (which it looks like it will be) you probabl need an `AcknowledgeFailure` here and before all transitions into the `Failure` state.

Reply:
I added two such states (I don't think more are necessary, apart from perhaps one before transition into the Success state).